ui/components/editblock/base/_index.scss

```diff
@@ -303,3 +303,123 @@
         height: 3rem;
     }
 }
+
+
+/**
+ * Inline Edit Block
+ */
+
+
+.nds-editblock_inline {
+    .nds-edit-block-edit__container {
+        border: 1px solid $color-border-input;
+        border-radius: 0.25rem;
+        padding: 2rem;
+        position: relative; 
+        width: 100%; 
+        z-index: 8888; 
+        background: white;
+
+        svg.nds-editblock_close {
+            fill: $color-border-input;
+            top: 0;
+            right: 0;
+            position: absolute;
+            cursor: pointer;
+        }
+
+        .nds-custom_button {
+            background: white;
+            padding: 0.2rem 3.5rem;
+            color: #0070d2;
```

Comment:
Hi @apurv3039 Sorry I should have said we need to do this for all hardcoded color values. These all need to be configurable from the design-tokens. So this #0070d2 needs to be changed.